src/redisai.c

```diff
@@ -721,6 +724,17 @@ struct RedisAI_RunInfo {
   RAI_Error* err;
 };
 
+struct RedisAI_RunStats {
```

Comment:
keeping track of the error counts that ocurred during backend running might be of use. what do you think @lantiga ?
please see https://github.com/tensorflow/serving/issues/462#issuecomment-367594255

Reply:
Yes they would, noted

Reply:
Done